Gemfile.lock

```diff
@@ -237,72 +238,72 @@ GEM
       timeout
     net-protocol (0.1.3)
       timeout
-    net-scp (3.0.0)
-      net-ssh (>= 2.6.5, < 7.0.0)
+    net-scp (4.0.0.rc1)
+      net-ssh (>= 2.6.5, < 8.0.0)
     net-smtp (0.3.1)
       digest
       net-protocol
       timeout
-    net-ssh (6.1.0)
+    net-ssh (7.0.1)
     nio4r (2.5.8)
-    nokogiri (1.13.3-x86_64-darwin)
+    nokogiri (1.13.8-x86_64-darwin)
       racc (~> 1.4)
     orm_adapter (0.5.0)
     pagy (5.10.1)
       activesupport
     parallel (1.22.1)
-    parser (3.1.1.0)
+    parser (3.1.2.1)
       ast (~> 2.4.1)
-    pg (1.3.5)
+    pg (1.4.3)
     propshaft (0.6.4)
       actionpack (>= 7.0.0)
       activesupport (>= 7.0.0)
       rack
       railties (>= 7.0.0)
-    public_suffix (4.0.6)
-    puma (5.6.4)
+    public_suffix (5.0.0)
+    puma (5.6.5)
       nio4r (~> 2.0)
     racc (1.6.0)
-    rack (2.2.3)
-    rack-test (1.1.0)
-      rack (>= 1.0, < 3)
-    rails (7.0.2.3)
-      actioncable (= 7.0.2.3)
-      actionmailbox (= 7.0.2.3)
-      actionmailer (= 7.0.2.3)
-      actionpack (= 7.0.2.3)
-      actiontext (= 7.0.2.3)
-      actionview (= 7.0.2.3)
-      activejob (= 7.0.2.3)
-      activemodel (= 7.0.2.3)
-      activerecord (= 7.0.2.3)
-      activestorage (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    rack (2.2.4)
```

Comment:
### [rack](https://github.com/rack/rack)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.2.3 → 2.2.4

[_[change-log](https://github.com/rack/rack/blob/master/CHANGELOG.md), [source-code](https://github.com/rack/rack), [gem-diff](https://my.diffend.io/gems/rack/2.2.3/2.2.4)_]

:tada: Patched vulnerabilities:

- [CVE-2022-30122](https://nvd.nist.gov/vuln/detail/CVE-2022-30122)
  **Denial of Service Vulnerability in Rack Multipart Parsing**

  
  URL: https://groups.google.com/g/ruby-security-ann/c/L2Axto442qk

  There is a possible denial of service vulnerability in the multipart parsing component of Rack.  This vulnerability has been assigned the CVE identifier CVE-2022-30122.  Versions Affected:  >= 1.2 Not affected:       < 1.2 Fixed Versions:     2.0.9.1, 2.1.4.1, 2.2.3.1  ## Impact Carefully crafted multipart POST requests can cause Rack's multipart parser to take much longer than expected, leading to a possible denial of service vulnerability.  Impacted code will use Rack's multipart parser to parse multipart posts.  This includes directly using the multipart parser like this:  ``` params = Rack::Multipart.parse_multipart(env) ```  But it also includes reading POST data from a Rack request object like this:  ``` p request.POST # read POST data p request.params # reads both query params and POST data ```  All users running an affected release should either upgrade or use one of the workarounds immediately.  ## Workarounds There are no feasible workarounds for this issue.


- [CVE-2022-30123](https://nvd.nist.gov/vuln/detail/CVE-2022-30123)
  **Possible shell escape sequence injection vulnerability in Rack**

  
  URL: https://groups.google.com/g/ruby-security-ann/c/LWB10kWzag8

  There is a possible shell escape sequence injection vulnerability in the Lint and CommonLogger components of Rack.  This vulnerability has been assigned the CVE identifier CVE-2022-30123.  Versions Affected:  All. Not affected:       None Fixed Versions:     2.0.9.1, 2.1.4.1, 2.2.3.1  ## Impact Carefully crafted requests can cause shell escape sequences to be written to the terminal via Rack's Lint middleware and CommonLogger middleware.  These escape sequences can be leveraged to possibly execute commands in the victim's terminal.  Impacted applications will have either of these middleware installed, and vulnerable apps may have something like this:  ``` use Rack::Lint ```  Or  ``` use Rack::CommonLogger ```  All users running an affected release should either upgrade or use one of the workarounds immediately.  ## Workarounds Remove these middleware from your application






<details>
<summary>Commits</summary>

A change of **18** commits. See the full changes on [the compare page](https://github.com/rack/rack/compare/2.2.3...2.2.4).

These are the first 10 commits:
- (547e809) [Don't ary.inspect in the lint assertions (backport) (#1765)](https://github.com/rack/rack/commit/547e809a9699ea48f7135b35857008ef4f37838a)
- (a2091fa) [Use custom exception on params too deep error.](https://github.com/rack/rack/commit/a2091fa40dc85b98e4de7626eb3d9891c693fbb9)
- (59d4440) [Merge pull request #1838 from RubyElders/custom-range-except…](https://github.com/rack/rack/commit/59d44405794a34aec7a34f3cef21bf1bedeaa5e7)
- (43b5565) [Newer rubies spec compatibility.](https://github.com/rack/rack/commit/43b5565a73817d66b6d96de2e28d525a2a56f852)
- (031b47a) [Replace CircleCI with GitHub Actions.](https://github.com/rack/rack/commit/031b47a8d5bd7b7b2416b751f522acfd7a5af036)
- (7ba69ba) [Merge pull request #1839 from RubyElders/2-2-stable-ci](https://github.com/rack/rack/commit/7ba69ba714b04adef191919552da63ec84fa1590)
- (991e2a5) [Add Ruby 2.3 compatibility for tests, add Ruby 2.3 to CI. (#…](https://github.com/rack/rack/commit/991e2a5083f1e2edf7e8b05bf026a17673bb27e7)
- (e71d214) [Ensure Rack::QueryParser::ParamsTooDeepError is inherited fr…](https://github.com/rack/rack/commit/e71d214f8573625b6f832cb9edd2aac898446e53)
- (a872bcc) [Restrict broken mime parsing](https://github.com/rack/rack/commit/a872bcce127a6f4a0f16af8f28b686955ba32814)
- (434413f) [Escape untrusted text when logging](https://github.com/rack/rack/commit/434413f5cff1ca36d70c3e5a3544c5c2d7b636bf)


</details>